src/inspector/main_thread_interface.cc

```diff
@@ -114,7 +115,7 @@ class AnotherThreadObjectReference {
   ~AnotherThreadObjectReference() {
     // Disappearing thread may cause a memory leak
     thread_->Post(
-        std::unique_ptr<DeleteRequest>(new DeleteRequest(object_id_)));
+        std::make_unique<DeleteRequest>(object_id_));
```

Comment:
It seems like this could be un-wrapped to a single line.

Reply:
Yes. I have some trouble set clang-format on wsl. I will fix it manually.